release.yaml

```diff
@@ -1,32 +1,149 @@
 rancher-monitoring:
+- 100.0.0+up16.6.0
```

Comment:
we need to remove all the extra entries from the release.yaml here

Reply:
updated